recipes/winflexbison/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class WinflexbisonConan(ConanFile):
+    name = "winflexbison"
+    description = "Flex and Bison for Windows"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lexxmark/winflexbison"
+    topics = ("conan", "winflexbison", "flex", "bison")
+
+    generators = "cmake"
+    license = "GPL-3.0-or-later"
+    exports_sources = ["CMakeLists.txt"]
+
+    settings = "os_build", "build_type", "arch", "compiler"
+
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
```

Comment:
Granted, it looks like it isn't needed because you aren't calling `cmake.install()` in the `package()` method.
But this makes this recipe look like all others.